src/ol/tile.js

```diff
@@ -44,6 +44,18 @@ ol.Tile = function(tileCoord, state) {
    */
   this.state = state;
 
+  /**
+   * An "interim" tile for this tile. The interim tile may be used while this
+   * one is loading, for "smooth" transitions when changing dynamic params.
+   * @type {ol.Tile}
+   */
+  this.interimTile = null;
+
+  /**
+   * @type {string}
```

Comment:
I've added more docs. I hope it clarifies a bit, but I am conscious it's not perfect.